src/main/proto/tech/figure/servicing/v1beta1/loan_state.proto

```diff
@@ -17,6 +17,13 @@ message LoanStructureState {
   repeated LoanState loan_states    = 2; // Loan state data for loans included in the structure
 }
 
+/**
+A submission of loan state metadata for a single loan
+ */
+message LoanStateSubmission {
+  repeated LoanStateMetadata loan_state = 1; // Loan state metadata for the loan
+}
```

Comment:
- Is there a suggestion for a better name?
- Is this sufficient documentation?

Reply:
I'm good with name + doc update